docs/en_US/Overview.md

```diff
@@ -46,6 +46,38 @@ For each experiment, user only needs to define a search space and update a few l
 
 More details about how to run an experiment, please refer to [Get Started](Tutorial/QuickStart.md).
 
+## Core Capacities
+
+Hyperparameter tuning is a core capacity of NNI, it can be used to tune hyperparameters in machine learning and deep learning models, parameters in complex systems, etc. While hyperparameter tuning is only one step in designing a machine learning or deep learning model. Thus, NNI is evolved to help more on finding a better model with the following (pipelined) components.
+
```

Comment:
why the `parameter tuning` is blue and others are green?

Reply:
thanks, will update this figure

Reply:
removed the figure